client/signup/step-wrapper/index.jsx

```diff
@@ -211,9 +215,9 @@ class StepWrapper extends Component {
 			sticky = isSticky;
 		}
 
-		const queryParams = new URLSearchParams( window?.location.search );
-		const flags = queryParams.get( 'flags' );
-		const isHelpCenterLinkEnabled = flags === 'signup/help-center-link';
```

Comment:
This change was required for testing with multiple flags enabled, such as `/setup/onboarding?flags=onboarding/force-goals-first,signup/help-center-link`.

For internal testing after PRs have been merged, we can use https://wpcalypso.wordpress.com/

Reply:
This is OK, but I have to mention that these two are not the same. @escapemanuele moved to manual parsing because we wanted to strictly test in production. Because you only get Zendesk production in production, and flags don't work there. 

With this change, you won't be able to test the whole experience. Even if you deploy to prod.

Reply:
Yes, I think we need to have flags checked this ways, otherwise we aren't able to test DIFM and Zendesk in production, as Omar said!

Reply:
Thanks, I have reverted this and changed it to support the case when multiple flags are passed.